graphrag/config/defaults.py

```diff
@@ -93,7 +93,7 @@
 VECTOR_STORE = f"""
     type: {VectorStoreType.LanceDB.value}
     db_uri: '{(Path(STORAGE_BASE_DIR) / "lancedb")!s}'
-    collection_name: default
+    container_name: default # A preffix for the vector store to create embbeding containers. Default: 'default'. For example 'default-entity-description'
```

Comment:
"entity-description" should not be part of the container, as that is added as a specific embedding collection name. I would just remove the example or use something generic like "myproject".

Reply:
I was trying to show an example of the prefix being used, but, I'll just remove the example all along to avoid confusion